src/pyOpenMS/pxds/ISpectrumAccess.pxd

```diff
@@ -5,7 +5,7 @@ from libcpp.vector cimport vector as libcpp_vector
 from OpenSwathDataStructures cimport *
 
 # ctypedef shared_ptr[ISpectrumAccess] SpectrumAccessPtr
-cdef extern from "<OpenMS/ANALYSIS/OPENSWATH/OPENSWATHALGO/DATAACCESS/ISpectrumAccess.h>" namespace "OpenSwath":
+cdef extern from "<OpenMS/OPENSWATHALGO/DATAACCESS/ISpectrumAccess.h>" namespace "OpenSwath":
```

Comment:
@timosachsenberg we change ISpectrumAccess here, but only the file location changed